imageio/core/v3_plugin_api.py

```diff
@@ -35,6 +37,7 @@ class ImageProperties:
 
     shape: Tuple[int, ...]
     dtype: np.dtype
+    n_images: Optional[int]
```

Comment:
```suggestion
    n_images: Optional[int] = None
```

Saves us from setting `None` when reading single images.